spec/models/proposal_spec.rb

```diff
@@ -67,6 +67,13 @@
     end
   end
 
+  describe "#video_url" do
+    it "should not be valid when URL is not from Youtube or Vimeo" do
```

Comment:
Really great to have test for it 👍 💯 . What do you think about testing valid scenarios? Maybe looping through the valid video domains array checking the helper `valid_video_url?` method validates them all?

Reply:
Just for clarification: You mean replacing this test with one where the `valid_video_url?` method tests each domain in the `valid_video_domains` array?

Reply:
Not replacing, but increasing. I mean a test should both check valid and invalid scenarios, right? And in this particular case, the valid scenarios are only a few so checking all of them may not be too slow I think (but I may be wrong and just checking one could be enough)

Reply:
I think one example should do. I created a `.txt` file under `/fixtures/files` with 5 valid URLs to test all the use cases, but according to Knapsack, this is too slow.

Creating an array on the test could work, but looks ugly since we have all these Youtube/Vimeo links.

What do you think?